tabular/src/autogluon/tabular/models/tab_transformer/tab_transformer_model.py

```diff
@@ -527,6 +535,7 @@ def load(cls, path: str, reset_paths=False, verbose=True):
         
         3) Bug where HPO doesn't work when cuda is enabled.
         "RuntimeError: Cannot re-initialize CUDA in forked subprocess. To use CUDA with multiprocessing, you must use the 'spawn' start method"
+        Update: This will likely be fixed in a future change to HPO in AutoGluon.
         
         4) Enable output layer of TT model to be multiple fully connected layers rather than just a single
```

Comment:
since 4) is completed, you can remove this :)

Reply:
Got it!